java/src/main/java/com/mlt/converter/mvt/MvtUtils.java

```diff
@@ -50,6 +61,23 @@ public static JtsMvt decodeMvt2Fast(ByteArrayInputStream mvtTile) throws IOExcep
     return MvtReader.loadMvt(mvtTile, geometryFactory, new TagKeyValueMapConverter(true, ID_KEY));
   }
 
+  /* Use the java port of the vector-tile-js library created by Dane Springmeyer.
+   * To get realistic number and have a fair comparison in terms of the decoding performance,
+   * the geometries of the features have to be decoded.
+   * */
+  public static Map<String, VectorTileLayer> decodeMvtMapbox(byte[] mvtTile) throws IOException {
+    Pbf pbf = new Pbf(mvtTile);
+    VectorTile vectorTile = new VectorTile(pbf, pbf.length);
+    for (var layer : vectorTile.layers.values()) {
+      for (int i = 0; i < layer.length; i++) {
+        var feature = layer.feature(i);
+        var geometry = feature.loadGeometry();
+      }
+    }
+
+    return vectorTile.layers;
```

Comment:
I propose iterating all features in both the MVT and MLT benchmarks and returning a count of some kind.